sdk/lib/3rdparty/freetype/src/type1/t1gload.c

```diff
@@ -380,12 +387,9 @@
     T1_GlyphSlot            glyph = (T1_GlyphSlot)t1glyph;
     FT_Error                error;
 #ifdef __REACTOS__
-    T1_DecoderRec *decoder = malloc(sizeof(T1_DecoderRec));
-    if (!decoder)
-      return FT_THROW( Out_Of_Memory );
-/* Ugly but it allows us to reduce the diff */
-#define decoder (*decoder)
-    {
+    T1_DecoderRec          *decoder = malloc(sizeof(*decoder));
```

Comment:
Please restore the NULL-check + early return.

Reply:
Restored.